### PR TITLE
Change the Concrete KL to logit space to avoid underflow.

### PR DESCRIPTION
**Pull request by eelcovdw**
Madison et al. recommend using the logits of the concrete distribution as stochastic node, to avoid numerical underflow in the KL. Pytorch now supports this, so we only need to change the KL estimation to use the base LogitRelaxedBernoulli distribution instead of the RelaxedBernoulli.

### CODE REVIEW

**Comment by eelcovdw** [comment]
See PR #4
